docs/semantic-search.ipynb

```diff
@@ -449,372 +373,131 @@
-            "b2d1aae111264a3a96bd21251d62cd2d",
-            "686aec60ca454da099f981f459154b18",
-            "9a543168eafa428f8ea707d6866f73ec",
-            "fcc0ded8483c4af5ac074ff4a4e57f62",
-            "cfbf4ac4cc5e432e999f675c5041722e",
-            "e6a57ba24239454a8cf07e6869f8232d",
-            "8ee1147298c148db8b94f83719c0dcd9",
-            "41b24e0881fa460d86f16245a4f906cd",
-            "9465699406694de2a28e3907b4e22ddb",
-            "d0d2453b3a274c5395136f3ef9c3b483",
-            "4d6f734fe74140659624715611935f06",
-            "8211e0cd7117440f8570f65633ea77db",
-            "5264427cf7d64adc8c2690835771185f",
-            "47298aba73ad458da7c804460c24fd6c"
-          ]
-        },
-        "id": "Fqo_hMRZiubM",
-        "outputId": "de4b4489-6f66-4ea0-b373-df1b88260c7d"
-      },
-      "outputs": [
-        {
-          "data": {
-            "text/plain": [
-              "SentenceTransformer(\n",
-              "  (0): Transformer({'max_seq_length': 256, 'do_lower_case': False}) with Transformer model: BertModel \n",
-              "  (1): Pooling({'word_embedding_dimension': 384, 'pooling_mode_cls_token': False, 'pooling_mode_mean_tokens': True, 'pooling_mode_max_tokens': False, 'pooling_mode_mean_sqrt_len_tokens': False, 'pooling_mode_weightedmean_tokens': False, 'pooling_mode_lasttoken': False, 'include_prompt': True})\n",
-              "  (2): Normalize()\n",
-              ")"
-            ]
-          },
-          "execution_count": 10,
-          "metadata": {},
-          "output_type": "execute_result"
-        }
-      ],
-      "source": [
-        "from sentence_transformers import SentenceTransformer\n",
-        "import torch\n",
+        "Now that our index is populated we can begin making queries. \n",
         "\n",
-        "device = 'cuda' if torch.cuda.is_available() else 'cpu'\n",
+        "The tricky part about querying with semantic search is that we'd normally need to involve an embedding model here again too! \n",
         "\n",
-        "model = SentenceTransformer('sentence-transformers/all-MiniLM-L6-v2', device=device)\n",
-        "model"
-      ]
-    },
-    {
-      "cell_type": "markdown",
-      "metadata": {
-        "id": "MP2-unZ--XJ9"
-      },
-      "source": [
-        "Now let's use this model to embed our question and find similar questions."
-      ]
-    },
-    {
-      "cell_type": "code",
-      "execution_count": 11,
-      "metadata": {
-        "colab": {
-          "base_uri": "https://localhost:8080/"
-        },
-        "id": "JWcO7jAK-N_1",
-        "outputId": "2bd676cb-a042-4695-e0d4-5ffb3b019f07"
-      },
-      "outputs": [],
-      "source": [
-        "def find_similar_questions(question):\n",
-        "    # Embed the question into a query vector\n",
-        "    xq = model.encode(question).tolist()\n",
+        "But with Pinecone's integrated infernece, we can just invoke our index we created and send the text we want to search with there. Neat!\n",
```

Comment:
Typo: "infernece"

Reply:
I'd elaborate on what actually happens here - the query text is converted into an embedding so that search compares vector to vector.